build.sbt

```diff
@@ -80,17 +80,17 @@ addCommandAlias("gemctl", "ctl/runMain gem.ctl.main")//
 resolvers in ThisBuild +=
   Resolver.sonatypeRepo("snapshots")
 
-// Before printing the prompt check git to make sure all is well.
-shellPrompt in ThisBuild := { state =>
-  if (version.value != imageManifest.formatVersion) {
-    import scala.Console.{ RED, RESET }
-    print(RED)
-    println(s"Computed version doesn't match the filesystem anymore.")
-    println(s"Please `reload` to get back in sync.")
-    print(RESET)
-  }
-  "> "
-}
+// // Before printing the prompt check git to make sure all is well.
+// shellPrompt in ThisBuild := { state =>
+//   if (version.value != imageManifest.formatVersion) {
+//     import scala.Console.{ RED, RESET }
+//     print(RED)
+//     println(s"Computed version doesn't match the filesystem anymore.")
+//     println(s"Please `reload` to get back in sync.")
+//     print(RESET)
+//   }
+//   "> "
+// }
```

Comment:
I think this is unnecessary and it's bothering me so I turned it off for now. Will revisit soon for the continuous deployment task.